ray-on-gke/platform/modules/kubernetes/variables.tf

```diff
@@ -35,3 +35,9 @@ variable "enable_autopilot" {
   description = "Set to true to enable GKE Autopilot clusters"
   default     = false
 }
+
+variable "enable_tpu" {
+  type = bool
+  description = "Set to true to create TPU node pool"
```

Comment:
Perhaps we should explain here that only one accelerator pool (GPU or TPU) will be created. Does Terraform support an enum value type?